py-polars/polars/expr/expr.py

```diff
@@ -5005,9 +5005,11 @@ def eq(self, other: Any) -> Self:
 
     def eq_missing(self, other: Any) -> Self:
         """
-        Method equivalent of equality operator `expr == other` where `None == None`.
+        Equality operator where `None` is treated as a distinct value.
```

Comment:
We should be talking about nulls instead of `None` in this context.

```suggestion
        Equality operator where null is treated as a distinct value.
```

Reply:
Fair point. I searched the docs and they largely follow the convention of using "null" when discussing values in the dataframe. Probably `None` was used initially because that's how you assign a value of null from Python, so I can imagine Python users who read pages needing some thought to realize how null arises.